### PR TITLE
Add google assistant compatibility for consultation (#14)

### PR DESCRIPTION
**Pull request by guillaumeNaimi**

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -6,42 +6,27 @@ logging:
...
+user:
+  username: ${ASTON_PARKING_ACTIONS_USERNAME:googleActionsUser}
+  password: ${ASTON_PARKING_ACTIONS_PASSWORD:vv9OLtNg8K9hZGZmA60sv47F}
 
+spring:
 # MONGO
   data:
     mongodb:
-      uri: mongodb://${ASTON_PARKING_MONGO_URI}
+      uri: mongodb://${ASTON_PARKING_MONGO_URI:astonparking:O0x2oYi7xLMX0GFk1AzqrZ1uz2FaUL78N0sfgAsnQu@192.168.57.30:27017/astonparking}
```
Comment (PierreLeresteux) [inline src/main/resources/application.yaml]: NOPE !!

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -6,42 +6,27 @@ logging:
...
 ##########################
 # SECURITY
 slack:
   client:
-    clientId: ${ASTON_PARKING_CLIENTID}
-    clientSecret: ${ASTON_PARKING_CLIENTSECRET}
+    clientId: ${ASTON_PARKING_CLIENTID:test}
```
Comment (PierreLeresteux) [inline src/main/resources/application.yaml]: Need a default value ?

**Review by PierreLeresteux — COMMENTED** [review]
1 critical leak found

File: src/main/resources/application.yaml (hunk 1 above)
Comment (guillaumeNaimi) [reply src/main/resources/application.yaml]: Didn't see I pushed that. I Removed the default value

File: src/main/resources/application.yaml (hunk 2 above)
Comment (guillaumeNaimi) [reply src/main/resources/application.yaml]: I thought I need it to run locally but I didn't need so I removed default values for slack

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -6,32 +6,18 @@ logging:
...
-            smtp:
-              starttls:
-                enable: false
-              auth: false
-              connectiontimeout: 5000
-              timeout: 5000
-              writetimeout: 5000
+google.actions:
+  username: ${ASTON_PARKING_ACTIONS_USERNAME:googleActionsUser}
+  password: ${ASTON_PARKING_ACTIONS_PASSWORD:vv9OLtNg8K9hZGZmA60sv47F}
```
Comment (PierreLeresteux) [inline src/main/resources/application.yaml]: same here for password !!!

**Review by PierreLeresteux — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yaml (hunk 3 above)
Comment (guillaumeNaimi) [reply src/main/resources/application.yaml]: removed

**Review by PierreLeresteux — APPROVED** [review]

**Comment by PierreLeresteux** [comment]
LGTM

**Pull request merged**